src/views/Form/Form.jsx

```diff
@@ -145,11 +125,34 @@ class Form extends React.Component {
         <div id="info">
           <Section section={params.section} subsection={subsection} />
           <SavedIndicator interval="30000" />
-          <TimeoutWarning timeout={env.SessionTimeout()} />
+          {showSessionWarning && <TimeoutWarning timeout={env.SessionTimeout()} />}
         </div>
       </div>
     )
   }
 }
 
+// TODO for some reason adding these is breaking this page (:
```

Comment:
Does this matter? Should we just delete it?

Reply:
I don't want to delete it - components should have propTypes and defaultProps defined. So this is a TODO left to flag this file for further attention (but is unrelated to this PR).